deployments/generic/kubearmor.yaml

```diff
@@ -226,6 +233,8 @@ spec:
         - --v=10
         image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
         name: kube-rbac-proxy
+        securityContext:
+          readOnlyRootFilesystem: true
```

Comment:
Let's not directly update YAML files here.

Can you make changes as part of https://github.com/kubearmor/KubeArmor/tree/main/deployments/get? And then run `make gen` should update all the respective deployments.

Also for the test scripts, Can you make these changes in `KubeArmor/KubeArmor/build/*.yaml` as well?

Reply:
done